modules/nf-core/sentieon/dedup/main.nf

```diff
@@ -60,6 +61,9 @@ process SENTIEON_DEDUP {
 
     sentieon driver $args $input_list -r ${fasta} --algo LocusCollector $args2 --fun score_info ${prefix}.score
     sentieon driver $args3 -t $task.cpus $input_list -r ${fasta} --algo Dedup $args4 --score_info ${prefix}.score --metrics ${metrics} ${prefix}${suffix}
+    # This following tsv-file is produced in order to get a proper tsv-file with Dedup-metrics for importing in MultiQC as "custom content".
+    # It should be removed once MultiQC has a module for displaying Dedup-metrics.
+    head -3 ${metrics} > ${metrics}.multiqc.tsv
```

Comment:
I'm thinking of pushing that to modules actually, but that can wait

Reply:
Sure, I can put it in the modules-repo if you prefer that.  Should I do that now?

Reply:
Yeah, let's go for it